app/services/image_service.rb

```diff
@@ -0,0 +1,25 @@
+module ImageService
+  class << self
```

Comment:
Why is this a class method?

Reply:
While doing it, I came across two ways of writing services. 
1. Service Objects as modules
2. Service Objects as classes
From which I tried [1] approach.